### ISSUE TITLE
Create an rc.d script to run stubby as a daemon

### ISSUE THREAD

**Issue by TELunus** [getdnsapi/stubby#93]
The documentation says to run stubby as `sudo stubby`, however this is an inconvenient way to run a system process that should always be left running, and that should come up with the system.  I see that support is being or has been added to run it using systemd.  As a user of FreeBSD I would also like to see a script for use with the rc system.

**Comment by saradickinson** [comment]
We have basic support for using systemd - see the systemd folder https://github.com/getdnsapi/stubby/tree/develop/systemd but we also have a couple of open issues on this #62 and #38 we need to look at.
I've updated the README to include the reference above.

**Comment by saradickinson** [comment]
I do agree - I was just highlighting our current lack of support in this general area :-)
@wtoorop do you have anything we could use here?

**Comment by wtoorop** [comment]
Yes!  I'll discuss with Jaap.  I believe he also does the rc.d scripts for nsd, unbound and opendnssec.

**Comment by wtoorop** [comment]
No time sorry... this is in for the 0.2.4
